cljfmt/src/cljfmt/core.cljc

```diff
@@ -118,18 +120,24 @@
    :set "#{", :deref "@",  :reader-macro "#", :unquote "~"
    :var "#'", :quote "'",  :syntax-quote "`", :unquote-splicing "~@"})
 
-(defn- prior-string [zloc]
-  (if-let [p (zip/left zloc)]
-    (str (prior-string p) (n/string (z/node p)))
-    (if-let [p (zip/up zloc)]
-      (str (prior-string p) (start-element (n/tag (z/node p))))
-      "")))
+(defn- prior-line-string [zloc]
+  (loop [zloc     zloc
+         worklist '()]
+    (if-let [p (zip/left zloc)]
+      (let [s (str (n/string (z/node p)))]
+        (if-not (str/includes? s "\n")
```

Comment:
Right, the ClojureScript support. I'm still used to thinking of this as Clojure only.

Still, I think it's worth using a reader conditional in this place if it means we can work with more versions of Clojure, and Leiningen has only recently moved to 1.8.0, so people using older Leiningens with lein-cljfmt would run into dependency problems.